src/test/java/org/cryptomator/windows/keychain/KeychainAccessProviderTest.java

```diff
@@ -14,19 +13,20 @@
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.Objects;
 
 public class KeychainAccessProviderTest {
 
 	@BeforeAll
 	public static void setup(@TempDir Path tmpDir) {
 		Path keychainPath = tmpDir.resolve("keychain.tmp");
-		System.setProperty("cryptomator.keychainPath", keychainPath.toString());
+		System.setProperty("cryptomator.integrationsWin.keychainPaths", keychainPath.toString());
 	}
 
 	@Test
 	@DisplayName("WindowsProtectedKeychainAccess can be loaded")
 	public void testLoadWindowsProtectedKeychainAccess() {
-		Assumptions.assumeFalse(System.getProperty("cryptomator.keychainPath", "").isBlank());
+		Assertions.assertTrue(Objects.nonNull(System.getProperty("cryptomator.integrationsWin.keychainPaths")));
```

Comment:
There is `Assertions.assertNotNull(...)` if we wanted to check for null. However this check was not an assertion but rather an assumption to make sure only to run the test if the path has been set. Which we may want to remove entirely, since we set this property during setup.

Reply:
Removed in [dd1890d](https://github.com/cryptomator/integrations-win/pull/14/commits/dd1890d84927613edb5c0525e8cc83c8b74f185a)